api/api.py

```diff
@@ -220,10 +219,24 @@ def __init__(self, *args, **kwargs):
             if self.draft_model is not None
             else self.model.config.max_seq_length
         )
+        self.system_fingerprint = (
+            self.builder_args.device + type(self.builder_args.precision).__name__
+        )
```

Comment:
Let's add a comment that this field doesn't match the spec, but is populated

We'll fix in a separate PR